sources/packages/forms/src/form-definitions/exportfinancialreports.json

```diff
@@ -372,67 +372,66 @@
                       "labelPosition": "top",
                       "labelWidth": "",
                       "labelMargin": "",
-                      "displayInTimezone": "viewer",
-                      "useLocaleSettings": false,
-                      "allowInput": true,
-                      "format": "yyyy-MM-dd",
-                      "placeholder": "",
+                      "placeholder": "Select date",
                       "description": "",
                       "tooltip": "",
+                      "prefix": "",
+                      "suffix": "",
+                      "widget": {
+                        "type": "calendar",
+                        "allowInput": true,
+                        "clickOpens": true,
+                        "enableDate": true,
+                        "enableTime": false,
+                        "mode": "single",
+                        "noCalendar": false,
+                        "format": "yyyy-MM-dd",
+                        "dateFormat": "yyyy-MM-dd",
+                        "useLocaleSettings": false,
+                        "hourIncrement": 1,
+                        "minuteIncrement": 5,
+                        "time_24hr": false,
+                        "saveAs": "text",
+                        "locale": "en"
+                      },
+                      "inputMask": "",
+                      "displayMask": "",
+                      "allowMultipleMasks": false,
                       "customClass": "",
                       "tabindex": "",
+                      "autocomplete": "",
                       "hidden": false,
                       "hideLabel": false,
+                      "showWordCount": false,
+                      "showCharCount": false,
+                      "mask": false,
                       "autofocus": false,
+                      "spellcheck": true,
                       "disabled": false,
-                      "tableView": false,
+                      "tableView": true,
```

Comment:
Can you please explain? Did we intentionally made this tableView: true ? or in the  version of formio, when saved its defaulted to true? I think the use of tableview is used more for the tasks in formio.

Reply:
I did not change the tableView intentionally @guru-aot. These may be the difference in default property values between Date-time component and Input-text component.